packages/zoe/test/unitTests/contracts/test-coveredCall.js

```diff
@@ -16,12 +16,21 @@ const coveredCallRoot = `${__dirname}/../../../src/contracts/coveredCall`;
 const atomicSwapRoot = `${__dirname}/../../../src/contracts/atomicSwap`;
 
 test('zoe - coveredCall', async t => {
-  t.plan(11);
-  const { moolaKit, simoleanKit, moola, simoleans, zoe } = setup();
+  t.plan(13);
+  const {
+    moolaKit,
+    simoleanKit,
+    bucksKit,
+    moola,
+    simoleans,
+    bucks,
+    zoe,
+  } = setup();
 
   const makeAlice = async (timer, moolaPayment) => {
     const moolaPurse = await E(moolaKit.issuer).makeEmptyPurse();
```

Comment:
For future reference, nothing about this scenario should require *Purses* as opposed to simply Payments.

Reply:
Yup, this comes from our earlier test design when we were trying to imitate Alice as she would act in real life as much as possible.